{"timestamp":"2025-10-21T15:05:40.079373Z","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager","filename":"manager.py","lineno":179}
{"timestamp":"2025-10-21T15:05:40.079891Z","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/fetch_and_load_data_dag.py","logger":"airflow.models.dagbag.DagBag","filename":"dagbag.py","lineno":593}
{"timestamp":"2025-10-21T15:05:40.100252Z","level":"info","event":"Task instance is in running state","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.100404Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.100818Z","level":"info","event":"Current task name:run_dbt","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.100901Z","level":"info","event":"Dag name:fetch_load_dbt_pipeline","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.101449Z","level":"info","event":"Tmp dir root location: /tmp","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":78}
{"timestamp":"2025-10-21T15:05:40.102524Z","level":"info","event":"Running command: ['/usr/bin/bash', '-c', 'cd /opt/airflow/dbt && dbt run --profiles-dir .']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":88}
{"timestamp":"2025-10-21T15:05:40.112213Z","level":"info","event":"Output:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":99}
{"timestamp":"2025-10-21T15:05:40.113632Z","level":"info","event":"/usr/bin/bash: line 1: dbt: command not found","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":106}
{"timestamp":"2025-10-21T15:05:40.113847Z","level":"info","event":"Command exited with return code 127","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook","filename":"subprocess.py","lineno":110}
{"timestamp":"2025-10-21T15:05:40.114265Z","level":"error","event":"Task failed with exception","logger":"task","filename":"task_runner.py","lineno":972,"error_detail":[{"exc_type":"AirflowException","exc_value":"Bash command failed. The command returned a non-zero exit code 127.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1307,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":416,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/bash.py","lineno":232,"name":"execute"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-10-21T15:05:40.115382Z","level":"info","event":"Task instance in failure state","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.115732Z","level":"info","event":"Task start","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.115827Z","level":"info","event":"Task:<Task(BashOperator): run_dbt>","logger":"task.stdout"}
{"timestamp":"2025-10-21T15:05:40.115882Z","level":"info","event":"Failure caused by Bash command failed. The command returned a non-zero exit code 127.","logger":"task.stdout"}
